spec.bs

```diff
@@ -2239,3 +2239,25 @@ achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-geolocation.https.html
   /fenced-frame/permission-notification.https.html
 </wpt>
+
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
+
+The <a href=https://www.w3.org/TR/cssom-view-1>CSSOM View Module</a> defines scroll bubbling behavior
+through the [=scroll a target into view=] algorithm. Our specification modifies it to not allow
+scrolling a target into view if it is inside of a fenced frame.
+
+Modify the [=scroll a target into view=] algorithm to add 2 steps after step 1 that reads:
+
+2. If the <var ignore=''>target</var>-associated {{Document}}'s [=node navigable=]'s
+   [=navigable/traversable navigable=] is a [=fenced navigable container/fenced navigable=], terminate these steps.
+3. If the <var ignore=''>target</var>-associated {{Document}}'s [=Document/ancestor navigables=] contains
+   a [=navigable=] whose [=navigable/traversable navigable=] is a [=fenced navigable container/fenced navigable=],
+   terminate these steps.
+
+Modify the [=Document/run the scroll steps=] algorithm to add a check before step 1.1 that reads:
+
+1. If <var ignore=''>target</var> is a {{Document}} whose [=node navigable=]'s
+   [=navigable/traversable navigable=] is a [=fenced navigable container/fenced navigable=],
+   continue.
```

Comment:
Below these algorithms, add a link to any web platform tests here as we do elsewhere in this spec.

Reply:
We currently have browser tests that are not reflected in any WPT: https://source.chromium.org/chromium/chromium/src/+/main:content/browser/renderer_host/scroll_into_view_browsertest.cc;l=873;drc=4a33cee8668c3ff5f07f9953c7abb921895b9275. Should we attempt to convert those into WPTs?

Reply:
For now, since there's nothing to add, I'm going to resolve this conversation. I added something to the tracker to track migrating WPTs, and, once they're migrated, we can update the spec to include those tests.